packages/asc-web-common/api/user/index.js

```diff
@@ -13,6 +13,14 @@ export function login(userName, passwordHash) {
   });
 }
 
+export function thirdPartyLogin(SerializedProfile) {
```

Comment:
Rename thirdPartyLogin to LoginThirdParty 

Rename SerializedProfile to profile